docs/data/data-grid/demo/PopularFeaturesDemo.js

```diff
@@ -315,10 +335,14 @@ const columns = [
     type: 'singleSelect',
     valueOptions: ['Premium', 'Pro', 'Community'],
     renderCell: (params) => {
-      if (!params.value) {
-        return '';
+      switch (typeof params.value) {
+        case 'string':
+          return <PlanTag plan={params.value} />;
+        case 'number':
+          return <CustomSizeAggregationFooter value={params.value} />;
+        default:
+          return '';
```

Comment:
While this works in this specific use case, I think we should utilize `params.aggregation` here: https://deploy-preview-8617--material-ui-x.netlify.app/x/react-data-grid/aggregation/#custom-rendering

Reply:
```tsx
renderCell: (params: GridRenderCellParams<any, string>) => {
  if (params.aggregation) {
    return params.formattedValue;
  }
  if (!params.value) {
    return '';
  }
  return <PlanTag plan={params.value} />;
},
```